src/main/java/nopol_examples/nopol_example_1/NopolExample.java

```diff
@@ -9,7 +9,7 @@
 	 */
 	public char charAt(String s, int index) {
 		
-		if ( index ==  0 ) // Fix index <= 0
+		if ( index == 0  )  // Fix index <= 0
```

Comment:
This line (12) has been identified with a suspiciousness value of 11.97%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_1.NopolExampleTest#test6`
- `nopol_examples.nopol_example_1.NopolExampleTest#test5`
</details>